src/java/org/apache/cassandra/db/virtual/AccordVirtualTables.java

```diff
@@ -327,7 +327,10 @@ private void process(SimpleDataSet ds, CommandStoreTxnBlockedGraph shard, Set<Tx
             CommandStoreTxnBlockedGraph.TxnState txn = shard.txns.get(txnId);
             if (txn == null)
             {
-                Invariants.checkState(reason == Reason.Self, "Txn %s unknown for reason %s", txnId, reason);
+                Invariants.checkState(reason != Reason.Self, "Self Txn %s is unknown", txnId);
```

Comment:
> 1) vtable showing what txn are blocking the queried table would throw error when txn isn’t known, which is valid (report historic transaction…)

  we have a concept of "historic transactions" and we now add a ton more (they used to be bootstrap only, now sync populates them), these are txn we *know* exist but don't know anything about them... `Self` is the txn the user asked to debug, so `!= Self` means this is a txn from a `Key` or direct dependency.  In both cases the `null` case "should" only be historic transactions, but throwing isn't useful so figured `<unknown>` is better